cirrus/server/cirrus/experiment_recipes.py

```diff
@@ -18,38 +18,62 @@ class RecipeType(Enum):
 
 
 class RemoteSettings:
-    def __init__(self, sdk: SDK):
-        self.recipes: dict[str, list[Any]] = {"data": []}
+    def __init__(self, sdk_live: SDK, sdk_preview: SDK):
```

Comment:
Okay actually, instead of having one RemoteSettings class that has parameters and attributes and methods for handling two collections, can't we just have two instances of RemoteSettings each of which only handles a single url, sdk, recipes, etc?